sources/debootstrap.go

```diff
@@ -25,7 +25,7 @@ func (s *debootstrap) Run() error {
 	release := strings.ToLower(s.definition.Image.Release)
 
 	// Enable merged /usr by default, and disable it for certain distros/releases
-	if distro == "ubuntu" && slices.Contains([]string{"xenial", "bionic"}, release) || distro == "mint" && slices.Contains([]string{"tara", "tessa", "tina", "tricia", "ulyana"}, release) || distro == "devuan" {
+	if distro == "ubuntu" && slices.Contains([]string{"xenial", "bionic"}, release) || distro == "debian" && slices.Contains([]string{"trixie", "sid"}, release) || distro == "mint" && slices.Contains([]string{"tara", "tessa", "tina", "tricia", "ulyana"}, release) || distro == "devuan" {
```

Comment:
The fact that this section is in lxd-imagebuilder and not in the distro recipe file is a bit of a code smell imho, distro specific logic shouldn't be in this tool really.

Reply:
I agree, we can introduce selected debootstrap flags/options in the config, but would need to think of an intuitive way to make certain options selective for the specific `downaloder`.

Currently downloader is global for entire distro, but we want to apply specific options only to some releases. Maybe something like this:
```
source:
  downloader: debootstrap
  
debootstrap:
  - options:
      merge-usr: true
    releases:
      - trixie
      - sid
```

Reply:
sounds good to me!

Reply:
Not sure what's going on with Debian as I removed this option not long ago: https://github.com/canonical/lxd-imagebuilder/pull/59

I'm expecting the next `base-files` update to fix the problem it introduced in 13.3 (13.2 was working without the deboostrap flag):

```
base-files (13.3) unstable; urgency=medium

  [ Helmut Grohne ]
  * DEP17: Install /usr-merge aliasing symlinks. Closes: #1064459.

 -- Santiago Vila <sanvila@debian.org>  Thu, 06 Jun 2024 00:35:00 +0200
```

We'll see how it goes.